### PR TITLE
Adds device 371050043 Paulmann Solar Light

### PR DESCRIPTION
**Pull request by AndyDorn**
Add for device 371050043 Paulmann Solar Light

### CODE REVIEW

File: src/devices/paulmann.ts [hunk 1]
```diff
@@ -171,6 +171,13 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Amaris LED panels',
         extend: [light({colorTemp: {range: undefined}, color: {modes: ['xy', 'hs']}})],
     },
+    {
+        zigbeeModel: ['371050043'],
+        model: '371050043',
+        vendor: 'Paulmann Licht',
```
Comment (Koenkk) [inline src/devices/paulmann.ts:177]:
```suggestion
        vendor: 'Paulmann',
```

**Comment by Koenkk** [comment]
Thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `371050043.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by AndyDorn** [comment]
Picture pull request is at https://github.com/Koenkk/zigbee2mqtt.io/pull/3034

**Comment by Great-Chart** [comment]
@ AndyDorn - Can I ask if you have this Paulmann Solely device yourself?
I note is is flagged as a router rather than an end device - its battery powered.  I'd like to see if with the battery level reported also; so I'll see what I can deduce once I have mine up and running conistently.

I found the pictorial instructions a little hard to fathom initially for pairing and inference is that the On/Off function only triggers the top pair of LEDs and presume that the lower one is triggered via a light sensor or internal PIR but uncertain how any such options are selected or varied.

Thus wondered what you have found out here (should you indeed have the device yourself)?

My database entry for reference is as below.

{"id":150,"type":"Router","ieeeAddr":"0x00158d000453986c","nwkAddr":62769,"manufId":4624,"manufName":"Paulmann Licht","powerSource":"DC Source","modelId":"371050043","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":256,"inClusterList":[0,4,3,6,5,4096],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"371050043","manufacturerName":"Paulmann Licht","powerSource":4,"zclVersion":3,"appVersion":11,"stackVersion":1,"hwVersion":1,"dateCode":"20190815","swBuildId":"1400-0004"}},"genOnOff":{"attributes":{"onOff":1}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00124b0024bc1862","endpointID":1}],"configuredReportings":[{"cluster":6,"attrId":0,"minRepIntval":0,"maxRepIntval":65000,"repChange":1}],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[33],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":11,"stackVersion":1,"hwVersion":1,"dateCode":"20190815","swBuildId":"1400-0004","zclVersion":3,"interviewCompleted":true,"meta":{"configured":332242049},"lastSeen":1729768696249}

**Comment by AndyDorn** [comment]
Yes, I own such device and run it in a test network but I was not able to test this further recently.
Anyway I am not quite satisfied for now.

My open (more or less negative) findings so far:
- device has battery but does not expose battery state
- once battery dies the device will leave the network silently (obviously) and occasionly rejoin once sun has reloaded the battery
- device acts as a router but I think it shoudn´t
- one light is controlled by some kind of illumination sensor at the front and this light and the sensor are not exposed or controllable
- another light can actually be switched on/off (but only lights on if the other light is on as well)
- when there is not enough sunlight during daytime to recharge the battery and the night is long the device will go offline
- if the controllable light is switched on it will even go offline sooner as it seems to drain the battery very much

the device should probably be reworkd by the manufacturer and get a larger battery or a secondary battery to make the network integration independent from the light power source.

my knowledge is limited but I am happy to help. cheers

**Pull request merged**